components/DoughnutGraph/index.jsx

```diff
@@ -0,0 +1,116 @@
+import Link from "next/link";
+import { Doughnut } from "react-chartjs-2";
+import { Chart, ArcElement, Tooltip } from "chart.js";
+import useSWR from "swr";
+import { Icon } from "@iconify/react";
+import Loading from "../Loading";
+import {
+  GraphContainer,
+  TotalContainer,
+  ListContainer,
+  ListItem,
+  ColorBox,
+  ItemName,
+  Amount,
+} from "@/design-system/StyledDoughnutGraph";
+
+Chart.register(ArcElement, [Tooltip]);
+
+function DoughnutGraph() {
+  const { data, error } = useSWR("/api/expenses");
+
+  if (!data) {
+    return <Loading></Loading>;
```

Comment:
this can be self-closing

```suggestion
    return <Loading />;
```